File: primerTrimestre/introduccion_programacion_python/1secuenciales_py/ej13sec.py
# Programa: ej13sec.py
# Propósito: Realizar un algoritmos que lea un número y que muestre su raíz cuadrada y su raíz cúbica.
# Python no tiene ninguna función predefinida que permita calcular la raíz cúbica, ¿cómo se puede calcular?
#
# Autor: Antonio Quesada
# Fecha: 13/10/2019.
#
# Variables a usar:
#   * num
#   * raiz_cuadrada
#   * raiz_cubica
#
# Algoritmo:
#   LEER num
#   raiz_cuadrada = num^1/2
#   raiz_cubica = num^1/3
#   ESCRIBIR raiz_cuadrada
#   ESCRIBIR raiz_cubica

import math

# Petición de datos
num = float(input("Inserte un número: "))
print("---------------------------------------------------------------")

#Cálculos
raiz_cuadrada = math.sqrt(num)
raiz_cubica = num**(1/3)

#Cálculos y Salida
print ("La raíz cuadrada de", num, "es", raiz_cuadrada)
print ("La raíz cúbica de", num, "es", raiz_cubica)





	
	

	






	
	

	

File: primerTrimestre/java basico/1secuenciales/Ej13sec.java
/** Realizar un algoritmos que lea un numero y que muestre su raiz cuadrada y su raiz cubica. 
 *  PSeInt no tiene ninguna funcion predefinida que permita calcular la raiz cubica, ¿como se puede calcular?.
 * 
 * @author Antonio Quesada
 */


import java.util.Scanner;
import java.math.*;

public class Ej13sec {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    //Definimos la variable.
    int num;
    
    //Leemos el numero.
    System.out.println("Introduce un numero: ");
    num = s.nextInt();
    
    //Raiz cuadrada usando la funcion Math.sqrt.
    System.out.println("La raiz cuadrada de "+num+" es "+Math.sqrt(num));
    
    //Raiz cubica elevando a 1/3 el numero.
    System.out.println("La raiz cubica de "+num+" es "+Math.pow(num,(double)1/3));
    
    

  }

}